test/integration/helpers.coffee

```diff
@@ -164,12 +164,24 @@ isProcessRunning = (pattern, callback) ->
   )
 
 
+kill = (pid, callback) ->
+  if process.platform is 'win32'
+    taskkill = spawn('taskkill', ['/F', '/T', '/PID', pid])
+    taskkill.on('close', -> callback())
+  else
+    try
+      process.kill(pid, 'SIGKILL')
```

Comment:
🤔 good question. The code isn't very helpful to explain why there's try/catch and the `# do nothing` comment isn't either. And it's quite important, actually. I'll add a comment or something.

Explanation: The `kill` test helper is meant to "kill if the process exists, do nothing if it doesn't exist". That's why there's no error handling for the `taskkill` subprocess, no return code checking, and why there's try/catch for the `process.kill()`, since it throws an error when the `pid` is a non-existent process.